libraries/fabric-shim/lib/stub.js

```diff
@@ -1007,6 +1007,29 @@ class ChaincodeStub {
         return this.handler.handleDeleteState(collection, key, this.channel_id, this.txId);
     }
 
+    /**
+     * purgePrivateData records the specified `key` to be purged in the private writeset of
+     * the transaction. Note that only hash of the private writeset goes into the
+     * transaction proposal response (which is sent to the client who issued the
+     * transaction) and the actual private writeset gets temporarily stored in a
+     * transient store. The `key` and its value will be purged from the collection
+     * when the transaction is validated and successfully committed.
```

Comment:
I don't think this is correct. The purge is a background process. See hyperledger/fabric-chaincode-go#55 for potential alternative wording. The descriptions should consistent once we've agreed what they should be anyway.